### PR TITLE
[CN-585] Update operator deployment step

### PR DESCRIPTION
**Pull request by cagric0**

### CODE REVIEW

File: docs/modules/ROOT/pages/get-started.adoc [hunk 1]
```diff
@@ -9,104 +9,48 @@ You need a Kubernetes or Openshift cluster, and the `kubectl` or `oc` command-li
 
 == Step 1. Deploy Hazelcast Platform Operator
 
-To deploy Hazelcast Platform Operator, apply the https://repository.hazelcast.com/operator/bundle-{full-version}.yaml[`bundle-{full-version}.yaml`] file to your Kubernetes cluster:
+Starting from the Hazelcast Platform Operator 5.6.0, a Helm chart is available to install the operator. It is available from the Hazelcast Helm Charts repository and can be added to your Helm repository list by running the following command:
```
Comment (angelasimms) [inline docs/modules/ROOT/pages/get-started.adoc:12]:
```suggestion
From release 5.6.0 onwards, you can use a Helm chart to install the Hazelcast Platform Operator. 

. Add the link:https://github.com/hazelcast/charts[Hazelcast Helm Charts repository] to your Helm repository list by running the following command:
```
I'd suggest add a number to each step in the process.

File: docs/modules/ROOT/pages/get-started.adoc [hunk 2]
```diff
@@ -9,104 +9,48 @@ You need a Kubernetes or Openshift cluster, and the `kubectl` or `oc` command-li
...
+helm repo add hazelcast https://hazelcast-charts.s3.amazonaws.com/
+helm repo update
 ----
---
-====
-
-TIP: If you always want to use the latest release, you can use the `bundle-latest.yaml` file.
 
-At this point, the Hazelcast Platform Operator should be up and running. You can check it with the command below.
+=== Deploy the Hazelcast Platform Operator and the CRDs together
```
Comment (angelasimms) [inline docs/modules/ROOT/pages/get-started.adoc:20]: We don't really use headings under steps. I'd suggest adding a numbered step and then two substeps, something like this...

. Deploy the Hazelcast Platform Operator. You can either deploy the Hazelcast Platform Operator at the same time as CRDs are separately. Deploying CRDs separately avoids installing any cluster-scoped resources.
.. Run the following command to.....
.. Run the following command to

**Review by angelasimms — CHANGES_REQUESTED** [review]
Hi, Could you reformat the content as in Step 1, with numbered points, so it follows the same format as Step 2, then I can review again?

File: docs/modules/ROOT/pages/get-started.adoc [hunk 3]
```diff
@@ -9,104 +9,57 @@ You need a Kubernetes or Openshift cluster, and the `kubectl` or `oc` command-li
...
-
-If you deploy multiple Hazelcast Operators in different namespaces, you need a way to differentiate the name of each operator. For example, you can give each operator's name a suffix by adding the `nameSuffix` field to the `kustomization.yaml` file:
 
-[source,yaml]
++
+. Monitor the operator logs
++
+At this point, the Hazelcast Platform Operator should be up and running. You can check it with the command below.
++
+[source,shell]
```
Comment (hasancelik) [inline docs/modules/ROOT/pages/get-started.adoc:58]: I think we should add an Openshift tab here, as we discussed.

**Review by hasancelik — CHANGES_REQUESTED** [review]

File: docs/modules/ROOT/pages/get-started.adoc (hunk 3 above)
Comment (cagric0) [reply docs/modules/ROOT/pages/get-started.adoc:58]: Sure, I forgot to re-add it.

File: docs/modules/ROOT/pages/get-started.adoc [hunk 4]
```diff
@@ -538,25 +491,17 @@ oc delete secret hazelcast-license-key
 
 Finally, run the command below to delete Hazelcast Platform Operator deployment.
```
Comment (angelasimms) [inline docs/modules/ROOT/pages/get-started.adoc:492]:
```suggestion
Finally, run the command below to delete the Hazelcast Platform Operator deployment.
```

**Review by angelasimms — COMMENTED** [review]
Just a few minor updates.

**Review by hasancelik — APPROVED** [review]

**Review by angelasimms — APPROVED** [review]
LGTM

**Pull request merged**